rules_gapic/gapic.bzl

```diff
@@ -56,19 +57,21 @@ gapic_srcjar = rule(
             allow_single_file = True,
             mandatory = True,
         ),
-        "gapic_yaml": attr.label(mandatory = True, allow_single_file = True),
-        "artifact_type": attr.string(mandatory = False, default = "GAPIC_CODE"),
-        "language": attr.string(mandatory = True),
+        "artifact_type": attr.string(mandatory = False), #default = "GAPIC_CODE"
+        "gapic_yaml": attr.label(mandatory = False, allow_single_file = True),
+        "language": attr.string(mandatory = False),
         "service_yaml": attr.label(mandatory = False, allow_single_file = True),
         "package_yaml2": attr.label(mandatory = False),
+        "package": attr.string(mandatory = False),
+        "output_suffix": attr.string(mandatory = False, default = ".srcjar"),
```

Comment:
nit: My intuition is that someone will specify a suffix without the period, like "jar" instead of ".jar". Feel free to challenge that

Reply:
The suffix is not always <dot>.<someting>. It could be anythin, sometimes empty, sometimes sometnhing like `"-set.proto.bin"` (check `proto_library_with_info` rule definition). Keeping it like this is necessary and makes the rule much more flexible.

Reply:
Totally makes sense. Are there docs for the calling interface?